recipes/aiida-lammps/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "aiida-lammps" %}
+{% set version = "0.4.0b3" %}
```

Comment:
Is there a non-beta / alpha version that can be released instead?

Reply:
Hey @scopatz, not yet I'm afraid. The package is written around `aiida-core` v1.0.0, which is on beta-3 release. The alpha release should be out soon hopefully.
Note the tests should be fixed once conda-forge/aiida-core-feedstock#15 is merged

Reply:
@scopatz the tests are now passing

Reply:
We generally don't package beta or alpha releases. How soon is soon?

Reply:
The final beta (b4) was released yesterday, and the stable release is scheduled for end of july

Reply:
Well is there not something in the 0.3 series?